packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -7,16 +7,15 @@ import {
   dataUiId,
   hidePiPiP,
   isTestProfileDesktop,
-  isTestProfileStableFlavor,
   pageClick,
   stableScreenshot,
   waitForSelector
 } from '../../common/utils';
 import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
 
 test.describe('Participant pane tests', async () => {
+  /* @conditional-compile-remove(one-to-n-calling) */
```

Comment:
nit: Possible for another PR, but this should include /* @conditional-compile-remove(PSTN-calls) */ as well.

Reply:
Oh yeah, sorry I should have rebased before tagging you. It's fixed in `main` from an earlier PR.